radar_control/radar_control.py

```diff
@@ -351,18 +367,18 @@ def send_datawrite_metadata(packet, radctrl_to_datawrite, datawrite_radctrl_iden
         if debug_samples:
             sequence_add.tx_data.txrate = debug_samples[sequence_index]['txrate']
             sequence_add.tx_data.txctrfreq = debug_samples[sequence_index]['txctrfreq']
-            sequence_add.tx_data.pulse_sequence_timing_us[:] = debug_samples[sequence_index][
-                'pulse_sequence_timing']
-            sequence_add.tx_data.pulse_offset_error_us[:] = debug_samples[sequence_index][
-                'pulse_offset_error']
+            sequence_add.tx_data.pulse_timing_us[:] = debug_samples[sequence_index][
```

Comment:
docstring for this function needs to be updated: param debug_samples info is outdated